packages/waku/src/minimal/client.ts

```diff
@@ -371,3 +371,79 @@ export const INTERNAL_ServerRoot = ({
     ...DEFAULT_HTML_HEAD,
     children,
   );
+
+// Use DIY `use` to workaround React SSR bug.
+// based on https://github.com/facebook/react/issues/29855
+// https://github.com/apollographql/apollo-client/blob/8e3edd4f5e5191453ba3ca1a1cc4fc4a02b936e8/src/react/hooks/internal/__use.ts#L15-L26
+
+export const use2: typeof use = (promise) => {
```

Comment:
This is not a public API, so add `INTERNAL_` prefix.
```suggestion
export const INTERNAL_use2: typeof use = (promise) => {
```